samples/net/zperf/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _zperf-sample:
+.. zephyr:code-sample:: zperfXXX
```

Comment:
NOTE: this xxx will go away as right now it's just a work-around until https://github.com/zephyrproject-rtos/zephyr/pull/62341 is merged. @jukkar you might even feel like reviewing that one? 😊